schema-engine/connectors/sql-schema-connector/src/warnings/view.rs

```diff
@@ -1,5 +1,8 @@
 use crate::introspection_pair::ViewPair;
-use schema_connector::{warnings as generators, Warnings};
+use schema_connector::{
```

Comment:
a little confused by this change, does this have any meaning outside semantics? and on that, what are the semantic benefits here